"kdotool missing or not available" on macOS

### Description

I'm unable to start espanso on macOS Tahoe after I unchecked "App Background Activity" (I didn't know that "Auca Coyan Maillot" equals "Espanso"...). Now, even if I re-enable it again, espanso is not working. Reinstalling does not help.

![Image](https://github.com/user-attachments/assets/b996056f-880e-40b1-9430-9105119dee7e)

### Steps to reproduce

1. download [latest ZIP](https://github.com/espanso/espanso/releases/latest/download/Espanso-Mac-Universal.zip) or install via `brew install espanso`
2. try to start espanso

### Expected behavior

Espanso running in the background. 

### Screenshots

_No response_

### Logs

```
00:35:41 [service(2253)] [INFO] reading configs from: "/Users/barabasz/.config/espanso"
00:35:41 [service(2253)] [INFO] reading packages from: "/Users/barabasz/.config/espanso/match/packages"
00:35:41 [service(2253)] [INFO] using runtime dir: "/Users/barabasz/Library/Caches/espanso"
00:35:41 [service(2253)] [INFO] system info: Darwin v26.0.1 - kernel: 25.0.0
00:35:41 [service(2253)] [INFO] kdotool missing or not available for the current wayland DE.
00:35:41 [service(2253)] [INFO] reloading espanso launchctl entry
00:35:41 [service(2253)] [INFO] service registered correctly!
00:36:08 [service(2365)] [INFO] reading configs from: "/Users/barabasz/.config/espanso"
00:36:08 [service(2365)] [INFO] reading packages from: "/Users/barabasz/.config/espanso/match/packages"
00:36:08 [service(2365)] [INFO] using runtime dir: "/Users/barabasz/Library/Caches/espanso"
00:36:08 [service(2365)] [INFO] system info: Darwin v26.0.1 - kernel: 25.0.0
00:36:08 [service(2365)] [INFO] kdotool missing or not available for the current wayland DE.
00:36:13 [service(2365)] [ERROR] unable to start service: timed out
00:36:13 [service(2365)] [ERROR] Hint: sometimes this happens because another Espanso process is left running for some reason.
00:36:13 [service(2365)] [ERROR]       Please try running 'espanso restart' or manually killing all Espanso processes, then try again.
```

### Your environment

- Operating system and version: Tahoe 26.0.1 
- Espanso version: 2.2.7


### Self-service

- [ ] I'd be willing to fix this bug myself.

--- a/espanso/src/main.rs
+++ b/espanso/src/main.rs
@@ -442,8 +442,7 @@ SubCommand::with_name("install")
         Ok(matches) => matches,
         Err(err) => match err.kind {
             ErrorKind::DisplayHelpOnMissingArgumentOrSubcommand | ErrorKind::DisplayHelp => {
-                clap_instance.print_help().expect("unable to print help");
-                std::process::exit(1);
+                err.exit();
             }
             ErrorKind::DisplayVersion => {
                 println!(

--- a/espanso/src/main.rs
+++ b/espanso/src/main.rs
@@ -20,7 +20,10 @@
 // This is needed to avoid showing a console window when starting espanso on Windows
 #![windows_subsystem = "windows"]
 
-use std::{path::PathBuf, process::Command};
+use std::path::PathBuf;
+
+#[cfg(target_os = "linux")]
+use std::process::Command;
 
 use clap::{App, AppSettings, Arg, ArgMatches, ErrorKind, SubCommand};
 use cli::{CliModule, CliModuleArgs};

--- a/espanso/src/main.rs
+++ b/espanso/src/main.rs
@@ -584,6 +584,7 @@ SubCommand::with_name("install")
         }
 
         // try to invoke `kdotool` to see if you have it or not.
+        #[cfg(target_os = "linux")]
         if Command::new("kdotool")
             .arg("getactivewindow")
             .arg("getwindowclassname")

